services/report/languages/xcode.py

```diff
@@ -70,12 +67,6 @@ def get_partials_in_line(line):
 
 
 def from_txt(content: bytes, report_builder_session: ReportBuilderSession) -> Report:
```

Comment:
one day codecov-rs will have samples and documentation for this so reviewing it isn't so difficult :P